src/colvartypes.h

```diff
@@ -1341,7 +1339,7 @@ class colvarmodule::rotation
   bool b_debug_gradients;
 
   /// \brief The rotation itself (implemented as a quaternion)
-  cvm::quaternion q;
+  cvm::quaternion q{1.0, 0.0, 0.0, 0.0};
```

Comment:
If I remember correctly the initialization in class declaration requires C++11, but we use C++11 anyway now so it would be OK.

Reply:
Yes, I'm enjoying the fact that we finally set this as a minimum! Hopefully VMD will move forward and we can adopt an even more modern standard :-)